nixos/modules/services/development/guix.nix

```diff
@@ -0,0 +1,118 @@
+{config, pkgs, lib, ...}:
+
+with lib;
+
+let
+
+  cfg = config.services.guix;
+
+  guix-binary = pkgs.stdenv.mkDerivation rec
+    { name = "guix-binary-${version}";
+      version = "0.16.0";
+
+      src = pkgs.fetchurl {
+        # TODO: make work on non x86_64 systems
+        url = "https://alpha.gnu.org/gnu/guix/guix-binary-${version}.x86_64-linux.tar.xz";
```

Comment:
Doesn't have `1.0.0` and also they recommend `https://ftp.gnu.org/gnu/guix/guix-binary-${version}.${pkgs.stdenv.targetPlatform.system}.tar.xz`.
I think that's the correct variable for the system string - not quite sure.

Reply:
Updated the URL. System string works (`$pkgs.stdenv.targetPlatform.system`). Also added correct SHA256s for platforms. However only tested on `x86_64-linux`.